openstack/rds/v3/backups/testing/requests_test.go

```diff
@@ -12,51 +12,6 @@ import (
 	"github.com/opentelekomcloud/gophertelekomcloud/testhelper/client"
```

Comment:
it is possible to move this into acceptance?

Reply:
done